pandas/tests/extension/test_arrow.py

```diff
@@ -783,11 +826,13 @@ def test_EA_types(self, engine, data, request):
             request.node.add_marker(
                 pytest.mark.xfail(raises=TypeError, reason="GH 47534")
             )
-        elif pa.types.is_timestamp(pa_dtype) and pa_dtype.tz is not None:
+        elif (
+            pa.types.is_timestamp(pa_dtype) and pa_dtype.tz is not None
+        ) or pa.types.is_decimal(pa_dtype):
             request.node.add_marker(
                 pytest.mark.xfail(
                     raises=NotImplementedError,
-                    reason=f"Parameterized types with tz={pa_dtype.tz} not supported.",
+                    reason=f"Parameterized types {pa_dtype} not supported.",
```

Comment:
#50689 fixes this for timestamptz, after both this and that are merged we can see about fixing this for decimal.